docs/installations/installation-on-gnu-linux.md

```diff
@@ -6,7 +6,7 @@ Installation on GNU/Linux
 For Debian-based Linux users, it is suggested to  use the [deb-package](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/) provided. It is most easily installed with **[gdebi](https://launchpad.net/gdebi)**
 which automatically resolves the dependencies available in the PPA Respositories.
 
-```
+```bash
```

Comment:
did this at least change anything in GitHub markdown for you ? 

(we can test a little before trying out on staging)

Reply:
Yes it does not highlight the "-" anymore. This will have more impact when the code blocks get more complex.